modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -9,6 +9,11 @@
 The `install-config.yaml` file requires some additional details.
 Most of the information is teaching the installer and the resulting cluster enough about the available hardware so that it is able to fully manage it.
 
+[NOTE]
+====
+The {op-system} image, `clusterOSImage`, used by the installer is no longer needed, as the correct ISO image is in the release payload.
```

Comment:
Note that `clusterOSImage` is/was a QCOW image. It's true that the one we get from the release payload is an ISO, but that is an implementation detail that users should never need to know, so it may be confusing to introduce it here.

Reply:
Updated as suggested for clarity and to remove "ISO".